src/play-item.js

```diff
@@ -0,0 +1,42 @@
+import {setup} from './auto-advance.js';
+
+/**
+ * Removes all remote text tracks from a player.
+ *
+ * @param  {Player} player
+ */
+const clearTracks = (player) => {
+  let tracks = player.remoteTextTracks() || [];
+
```

Comment:
Ah, gotcha, gotcha. Well, the tests pass, but that's using an array and not a true `TextTrackList`; so, I'll go back to a `while` here.